src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,618 @@
+        <p><input class="mx-2" type="checkbox">iPhone Cases & Clips</p>
+        <p><input class="mx-2" type="checkbox">TV & Home Theater</p>
+        <p><input class="mx-2" type="checkbox">Small Kitchen Appliances</p>
+        <div class="w-75 mb-3  border-bottom border-secondary divider"></div>
+        <h5 class="filters-caption">Brand</h5>
+        <p><input class="mx-2" type="checkbox">Insigni</p>
+        <p><input class="mx-2" type="checkbox">Samsung</p>
+        <p><input class="mx-2" type="checkbox">Apple</p>
+        <p><input class="mx-2" type="checkbox">HP</p>
+        <p><input class="mx-2" type="checkbox">Sony</p>
+        <p><input class="mx-2" type="checkbox">Metra</p>
+        <p><input class="mx-2" type="checkbox">Dyne</p>
+        <p><input class="mx-2" type="checkbox">LG</p>
+        <p><input class="mx-2" type="checkbox">Canon</p>
+        <p><input class="mx-2" type="checkbox">Speck</p>
+      </form>
+    </div>
+    <button class="border-0 rounded bg-primary shadow text-white py-2">CLEAR ALL FILTERS</button>
+  </div>
+
+  <div class=" col results d-flex flex-column gap-3">
+    <div class="d-flex justify-content-between caption">
+      <h5 class="">7,618 results found</h5>
+      <button class=" px-3 py-2 bg-primary rounded text-white border-0">
+        <svg class="bi bi-heart" fill="currentColor" height="16" viewBox="0 0 16 16" width="16"
+             xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="m8 2.748-.717-.737C5.6.281 2.514.878 1.4 3.053c-.523 1.023-.641 2.5.314 4.385.92 1.815 2.834 3.989 6.286 6.357 3.452-2.368 5.365-4.542 6.286-6.357.955-1.886.838-3.362.314-4.385C13.486.878 10.4.28 8.717 2.01L8 2.748zM8 15C-7.333 4.868 3.279-3.04 7.824 1.143c.06.055.119.112.176.171a3.12 3.12 0 0 1 .176-.17C12.72-3.042 23.333 4.867 8 15z"></path>
+        </svg>
+      </button>
+    </div>
+    <div class="d-flex flex-column gap-3 under-caption">
+      <form class="d-flex flex-row">
+        <input class="flex-grow-1 rounded-start border-0 shadow px-3 py-2" placeholder="Search" type="search">
+        <button class="bg-white rounded-end border-0 shadow px-3 py-2" type="submit">
+          <svg class="bi bi-search" fill="currentColor" height="16" viewBox="0 0 16 16" width="16"
+               xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
+          </svg>
+        </button>
+      </form>
+      <main class="container-fluid p-1 m-0">
+        <div class="row row-cols-1 row-cols-md-2 row-cols-xxl-3 g-4">
+          <div>
+            <section class="col card pt-4 border-0 shadow rounded d-flex flex-column justify-content-center">
+              <figure class="d-flex justify-content-center ">
+                <img alt="Apple Watch" class="w-75 product" src="images/product.png">
```

Comment:
consider to add a working image for product, this one is corrupted

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)